scripts/bundle.config.js

```diff
@@ -49,7 +49,9 @@ const BABEL_CONFIG = {
   plugins: [
     ['@babel/plugin-transform-runtime', {useESModules: false}], 
     'version-inline'
-  ]
+  ],
+  // Consider the file a "module" if import/export statements are present, or else consider it a "script".
+  sourceType: 'unambiguous'
```

Comment:
How does this help us? What is "the file" that is being considered a module here?

We do have a couple of script based examples do they fail to load without this change, and succeed to load with this change?

Reply:
Why is `modules:false` set? I don't think you want to preserve ES modules: https://babeljs.io/docs/en/babel-preset-env#modules

deck.gl's bundle settings: https://github.com/visgl/deck.gl/blob/master/scripts/bundle.config.js#L65

Reply:
@ibgreen @Pessimistress 

The problem - `dist.min.js:1 Uncaught TypeError: Cannot assign to read only property 'exports' of object '#<Object>'`caused by mixing `import` and `module.exports` syntax while code transpiling using `babel`.
The problem is described [here](https://github.com/webpack/webpack/issues/4039)

I tried manually replace `module.exports` to `export default` in `modulles/csv/src/bundle.js `file and it becomes work.
So As I understood we are having the options to fix this issue:
1) Use `umd` standard like this `['@babel/preset-env', {modules: 'umd'}]` - This is a common module format which will be supported as by `browser` as by `NodeJs`. - **The preferred fix!**
1) Replace all `module.exports`  with `export default` in all files in project.
2) Use `"sourceType": "unambiguous"` flag.  It is described [here](https://babeljs.io/docs/en/options#sourcetype)

Reply:
While testing in `examples/gallery` there are no errors on`2.3.9` version. Looks like not all modules are broken.
If I try to fetch another module for example `csv` loader in `examples/gallery/progress.html` on version `2.3.9` I got the same error: `dist.min.js:1 Uncaught TypeError: Cannot assign to read only property 'exports' of object '#<Object>'`
If I apply my changes and will try to load script locally from `modules/csv/dist/dist.min.js` there are no errors and `loaders` object exists and has `CSVLoader` inside.

![image](https://user-images.githubusercontent.com/70207219/105998896-22de9f80-60be-11eb-85ac-3ee85f90f96b.png)


![image](https://user-images.githubusercontent.com/70207219/105998946-2c680780-60be-11eb-936f-c6b42f80a4ac.png)